ggml/src/ggml-vulkan/ggml-vulkan.cpp

```diff
@@ -8592,6 +8628,14 @@ static void ggml_vk_op_f32(ggml_backend_vk_context * ctx, vk_context& subctx, co
             }
         }
         break;
+    case GGML_OP_SSM_CONV:
+        {
+            const uint32_t nr  = src0->ne[1];
+            const uint32_t n_t = dst->ne[1];
+            const uint32_t n_s = dst->ne[2];
+            elements = { CEIL_DIV(nr, 32), n_t, n_s };
```

Comment:
Seems like you could use wg_denoms for this one (make wg_denoms = {32, 1, 1} and not explicitly use CEIL_DIV here).